src/main/resources/application.yaml

```diff
@@ -154,6 +154,9 @@ features:
   federated-s-s-o: ${federated.sso:true}
   sso-auto-login-redirect: true
   step-up-authentication: ${step.up.authentication:true}
+  dynatrace:
+    monitor:
+      endpoint: "/rb_bf24054dsx"
```

Comment:
Is this the prod value?
Should we add the AAT one to `values.aat.template.yaml`?

Reply:
No, it's the non-prod value, all the non-prod environments are the same. I thought we'd put the prod value in flux?

Reply:
Ok, makes sense.